code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -89,6 +89,22 @@
 			NT.Insert(H, drop_if_replaced = FALSE)
 		else
 			tail.Remove(H)
+			
+/datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/M)
+	.=..()
+	if(chem.type == /datum/reagent/consumable/coco)
+		if(prob(40))
+			M.adjust_disgust(20)
+		if(prob(5))
+			M.visible_message("<span class='warning'>[M] [pick("dry heaves!","coughs!","splutters!")]</span>")
+		if(prob(10))
+			var/sick_message = pick("You feel nauseous.", "You're nya't feeling so good.","You feel like your insides are melting.","You feel illsies.")
+			to_chat(M, "<span class='notice'>[sick_message]</span>")
+		if(prob(15))
+			var/obj/item/organ/guts = pick(M.internal_organs)
+			guts.applyOrganDamage(15)
+		return TRUE

```

Comment:
THIS NEEDED TO BE A TAB BACKWARD! OH LORDE IS ME!!

Reply:
Just means it will process as usual too.